Package.swift

```diff
@@ -13,7 +13,8 @@ let package = Package(
             targets: ["FaceLiveness"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/aws-amplify/amplify-swift", exact: "2.27.2")
+        // TODO: Change this before merge to main
+        .package(url: "https://github.com/aws-amplify/amplify-swift", branch: "no-light-implementation")
```

Comment:
revert branch

Reply:
i can revert it in a follow up as it's needed for CI to build the changes in this PR.